_guides/client/basic.md

```diff
@@ -11,6 +11,16 @@ extern crate hyper;
 extern crate tokio_core;
 ```
 
+Make sure you have these dependencies listed in your ```Cargo.toml```, else you may get errors. You may need 
+to check if these are the latest versions from [Crates.io](https://crates.io)
+
+```
+`[dependencies`]
+hyper = "0.11"
+futures = "0.1.14"
+tokio-core = "0.1.8"
```

Comment:
If we just suggest "0.1" for both, Cargo will pick the latest version automatically.